src/Dockerfile-base

```diff
@@ -35,6 +35,8 @@ RUN conda install -y python=${python_version} scikit-learn six h5py \
         matplotlib pillow rasterio jupyter scikit-image && \
     conda clean -yt
 
+RUN pip install spectral
```

Comment:
Consider removing this line and just adding `spectral` to the next line which runs `pip install`.